homeassistant/components/splunk.py

```diff
@@ -0,0 +1,94 @@
+"""
+homeassistant.components.splunk
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+Splunk component which allows you to send data to an Splunk instance
+utilizing the HTTP Event Collector.
+"""
+import logging
+import requests
+import json
```

Comment:
The Python 3.4 Travis build is passing, but the 3.5 build is failing with the message:
```
Checking style with flake8...
Checking style with pylint...
************* Module homeassistant.components.splunk
C:  9, 0: standard import "import json" comes before "import requests" (wrong-import-order)
```
To my human eyes, it looks like this is not the case.  Can anyone explain why Travis sees this differently?